scripts/get_stats.py

```diff
@@ -0,0 +1,38 @@
+from glob import glob
+
+
+def get_stats(folder):
+    stats = {
+        "get": [],
+        "post": [],
+        "average_get": None,
+        "average_post": None,
+        "average_total": None,
+        "total_requests": 0,
+        "total_failures": 0,
+        "error_percentage": 0
+    }
+
+    for file in glob(folder + '/*stats.csv'):
+
+        with open(file) as f:
+            data = f.read()
+
+        for line in data.split('\n'):
+            if 'Name' in line:
+                continue
+
+            values = line.split(',')
+
+            percentile_99th = int(values[18])
```

Comment:
Is this getting the last entry in the list? If so, `[-1]` would be a safer approach

Reply:
No, I think there are 22 columns in the file 😐